### DISCUSSION TITLE
How to make GrapesJS dynamic?

### DISCUSSION THREAD

**Discussion by ghost** [GrapesJS/grapesjs#5976] [Q&A]
I think there are several solutions to this, the first one is to use the repository instead of the website webpage-builder also called the "if you just want to quickly build stuff use ...". Because im not an software developer and the other GitHub Repo's that offer pre-build Dynamic Grapesjs Frameworks are complete glibbish to me (i do not understand how to use them), i am stuck, i tried installing the GrapesJs Repository several times now and trust me i was crying because i was not able to configure it properly to make it a dynamic Builder (i am a hobby coder, this is out of my range of capabilities).

Just using the Website Page-builder was so easy, without any Problems, i was developing my Webpage feeling like a true WebDev  and guys hear me out, it actually made fun to build, because i didn't feel like i have to binchwatch FreeCodeAcademy to make a simple ass Webpage. 

**I ask for:** 
A simple grapesjs framework to enrich my current static Gallery sharing Platform into a more interactive one (i think you need .js and .php in order to do that). Is there a version of grapesjs out there that does not require manipulate Code structures; installing Plugins; API stuff, to make an Dynamic Webpage in the Drag and Drop (Soy) Way. 

Thanks for your time you spend on reading this.

**Comment by andrescevp** [comment]
https://github.com/GrapesJS/grapesjs/discussions/5445#discussioncomment-10922452

**Comment by artf** [comment]
@marlonivo did you try our official app [Studio](https://app.grapesjs.com/studio)? No need for account/subscription, build, export and publish wherever you want

**Reply by ghost** [reply]
This doesn't help at all, i think you misunderstood my questions, but thanks anyway.

**Comment by ghost** [comment]
Dynamic drag and drop Webpage Builder are not available for free in the moment, current closed source solutions are webflow which is heavily pay walled.

**Reply by artf** [reply]
I guess I'm still confused on what you're asking from GrapesJS 🥲
